lib/testing/src/lib/process-services/pages/form-fields.page.ts

```diff
@@ -148,8 +148,12 @@ export class FormFieldsPage {
         await BrowserActions.clearSendKeys(input, value);
     }
 
-    async isCompleteFormButtonDisabled(): Promise<string> {
-        await BrowserVisibility.waitUntilElementIsVisible(this.completeButton);
-        return this.completeButton.getAttribute('disabled');
+    async isCompleteFormButtonEnabled(): Promise<boolean> {
+        try {
```

Comment:
Same comment as above.

Reply:
Created another method to check for Complete button visibility.